portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,6 +31,8 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     private final DatastoreService datastore;
+    private final String inputName = "comment-input";
```

Comment:
nit: these two strings can be static.

Reply:
Also, the typical naming convention for static member variables is UPPER_CASE, so here it would be INPUT_NAME and DATASTORE_LABEL;

Reply:
Should the DatastoreService object also be all capitalized?